packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -530,7 +535,7 @@ describe('properties panel', function() {
               });
 
               // when
-              const input = screen.getByLabelText('Value', { selector: '#bio-properties-panel-Radio_1-value-0-value' });
+              const input = screen.getByLabelText('Value', { selector: '#bio-properties-panel-Radio_1-values-staticValue-0-value' });

```

Comment:
Is the ID change really needed or just a consequence of the re-arrangement of the entries?

It's not a breaking change as technical IDs are anyway just internal, but I'd like to prevent these changes (if not needed).

Reply:
I do want to keep these somewhat logical so I'll still rename but to something that makes more sense :)